src/components/Person/Person.jsx

```diff
@@ -1 +1,11 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am {person.age}</p>}
+    <p className="Person__partner">
+      {person.isMarried === true
+        ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+        : 'I am not married'}
```

Comment:
nested ternary looks complicated, let's create a separate variable for a person partner and use it here

`const partner = person.sex === 'm' ? 'wife' : 'husband'`

Reply:
dont know how to use it in react

Reply:
Not fixed